ui/public/locales/en.json

```
@@ -2185,6 +2206,13 @@
 "label.templatetype": "Template Type",
 "label.tftp.dir": "TFTP Directory",
 "label.tftpdir": "Tftp root directory",
+"label.threadsblockedcount": "Blocked Threads",
+"label.threadsdeamoncount": "Deamon Threads",
```

```suggestion
"label.threadsdeamoncount": "Daemon Threads",
```
key would need change as well